plugins/lando/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Lando ZSH (lando-zsh)
```

Comment:
The header goes in the first line, and always leave an empty newline after it (same for the rest of headers).